source/__tests__/integration/create-property.test.js

```diff
@@ -43,23 +35,82 @@ describe('Create profile has properties for resource and templates for property'
       //  to show a template can be selected
       await expect_sel_to_exist(`${template_select_sel}.ng-pristine`)
       await expect_sel_to_exist(`${template_select_sel} > option[selected="selected"][value="?"]`)
-      const value_selected = 'profile:bf2:Form'
-      await page.select(template_select_sel, value_selected)
+      await page.select(template_select_sel, 'profile:bf2:Form')
       await expect_sel_to_exist(`${template_select_sel}.ng-dirty`)
       await expect_sel_to_exist(`${template_select_sel} > option[selected="selected"][value^="profile:bf2"]`)
     })
   })
 
-  describe('property header appearence changed', () => {
-
+  describe('property header appearence', () => {
     it('font-awesome icon class is present', async () => {
       await expect_sel_to_exist(`.fa-caret-right`)
     })
+  })
+})
+
+describe('property URI is required', () => {
+  let profileFormSel = 'form[name="profileForm"]'
+  let exportButtonSel = 'a.btn.import-export'
+
+  // beforeEach(async () => {
+  //   await page.$eval(profileFormSel, e => e.reset(), {timeout: 1000})
+  // })
```

Comment:
should these commented out lines come out?  Or does it not matter?

Reply:
@jermnelson ^^   wanna decide?  Merge or not?

Reply:
I meant to remove those lines..